### PR TITLE
Feat/modal success error signup

### PR DESCRIPTION
**Pull request by danJecu**
# Description
Added modal for success and error for both signin and signup
# Jira task link
https://hackyourfuture-dk.atlassian.net/jira/software/projects/LANDING/boards/6?selectedIssue=LANDING-95
# Screenshots (if applicable)
image
image

### CODE REVIEW

File: packages/client/src/components/Auth/Signin.jsx [hunk 1]
```diff
@@ -16,17 +23,18 @@ export const Signin = () => {
...
+      setIsModalOpen(true);
+      setTimeout(() => {
+        navigate('/');
+      }, 1500);
     } catch (err) {
-      // error
       setError(err.message);
+      setIsModalOpen(true);
+      setTimeout(() => {
+        navigate('/signin');
```
Comment (shpomp) [inline packages/client/src/components/Auth/Signin.jsx:34]: Is this the /auth? Or should /auth be this? 
AFAIK /signin is currently a 404 page, right?

File: packages/client/src/components/Auth/Signin.jsx [hunk 2]
```diff
@@ -55,6 +63,20 @@ export const Signin = () => {
           <Link to="/signup"> Signup</Link>
         </Typography>
       </Form>
+
+      <Modal open={isModalOpen}>
+        {error ? (
+          <ModalBox backgroundColor="hoverRed">
+            <Typography variant="h5">
+              Something went wrong, please try again.
```
Comment (shpomp) [inline packages/client/src/components/Auth/Signin.jsx:71]: "The email address or password you entered is invalid" for the appropriate cases and the current message for everything else would be better UX 🙏🏽

**Review by shpomp — APPROVED** [review]
Overall nice and clean, just some nitpicks! 

An informative message modal on logout would be nice and consistent 🙏🏽

File: packages/client/src/components/Auth/Signin.jsx (hunk 1 above)
Comment (danJecu) [reply packages/client/src/components/Auth/Signin.jsx:34]: Changed the paths to match

File: packages/client/src/components/Auth/Signin.jsx (hunk 2 above)
Comment (danJecu) [reply packages/client/src/components/Auth/Signin.jsx]: Fixed!

**Comment by danJecu** [comment]
We will handle the logout in a new task 😁

**Pull request merged**